influence_benchmark/utils/wandb_logging.py

```diff
@@ -205,10 +206,35 @@ def print_stats_and_log_to_wandb(turns_df, traj_df, iteration_step, top_n, trajs
     print("====================")
 
     if log_to_wandb:
-        trajectories = get_trajs_wandb_html(turns_df)
-        # Shuffle the trajectories in the df
-        random.shuffle(trajectories)
-        for trajectory in trajectories[:trajs_to_log]:
+        top_turns_df: pd.DataFrame = get_logging_turns_df(
+            turns_df,
+            num_chosen_trajs=top_n_to_log,
+            func=pd.DataFrame.nlargest,
+        )
+
+        top_trajectories = get_trajs_wandb_html(top_turns_df)
+
+        for trajectory in top_trajectories:
+            wandb.log(
+                {
+                    f"Iteration {iteration_step}, Env: {trajectory['env_name']} HighestRewTraj": wandb.Html(
+                        trajectory["html_content"]
+                    )
+                }
+            )
+
+        bottom_turns_df: pd.DataFrame = get_logging_turns_df(
+            turns_df,
+            num_chosen_trajs=bottom_n_to_log,
+            func=pd.DataFrame.nsmallest,
+        )
+
+        bottom_trajectories = get_trajs_wandb_html(bottom_turns_df)
+        for trajectory in bottom_trajectories:
             wandb.log(
-                {f"Iteration {iteration_step}, Env: {trajectory['env_name']}": wandb.Html(trajectory["html_content"])}
+                {
+                    f"Iteration {iteration_step}, Env: {trajectory['env_name']} LowestRewTraj": wandb.Html(
+                        trajectory["html_content"]
+                    )
+                }
```

Comment:
```suggestion
                {f"Iteration {iteration_step}, Env: {trajectory['env_name']}": wandb.Html(trajectory["html_content"])}
```

Reply:
I think I prefer this way of logging that puts the bottom and top ones together in the same wandb panel. We already have far too many wandb panels